config.toml

```diff
@@ -303,6 +303,18 @@ gas_price = { price = 0.025, denom = 'stake' }
 # Minimum value: 1.0
 gas_multiplier = 1.1
 
+# Query the current gas price from the chain instead of using the static `gas_price` from the config.
+# Useful for chains which have [EIP-1559][eip]-like dynamic gas price. 
+#
+# At the moment, only chains which support the `osmosis.txfees.v1beta1.Query/GetEipBaseFee`
+# query can be used with dynamic gas price enabled.
+#
+# See this page in the Hermes guide for more information:
+# https://hermes.informal.systems/documentation/configuration/dynamic-gas-fees.html
+# 
+# Default: { enabled = false, gas_price_multiplier = 1.1, max_gas_price = 0.6 }
+dynamic_gas = { enabled = false, gas_price_multiplier = 1.1, max_gas_price = 0.6 }
```

Comment:
@ljoss17 We should probably name this `dynamic_gas_price` and change the format to 

```toml
dynamic_gas_price = { enabled = false, multiplier = 1.1, max = 0.6 }
```

What do you think?

Reply:
It would indeed be more accurate. I set it to `dynamic_gas` in gas there will be configurations related to dynamic gas but not directly the price.